content/en/docs/components/training/user-guides/prometheus.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Prometheus Monitoring"
+description = "Prometheus Metrics for the Training Operator"
+weight = 70
++++
+
+This guide explains how to monitor Kubeflow training jobs using Prometheus metrics. The Training Operator exposes these metrics, providing essential insights into the status of distributed machine learning workloads.
+
+{{< note >}}
+Metrics are only generated in response to specific events. For example, job creation metrics will only appear after a job has been created. If a metric is not visible, it may be because the corresponding event has not occurred yet.
+{{< /note >}}
+
+## Prometheus Metrics for Training Operator
+The Training Operator includes a built-in `/metrics` endpoint exposes Prometheus metrics. This feature is enabled by default and requires no additional configuration for basic use.
+
+### Configuring Metrics Port
+By default, metrics are exposed on port 8080. If you want to change the default port for metrics exporting, simply add the `metrics-bind-address` argument.
+
+For example, to change the metrics port to 8082:
+```yaml
+# deployment.yaml for the Training Operator
+spec:
+    containers:
+    - command:
+        - /manager
+        image: kubeflow/training-operator
+        name: training-operator
+        ports:
+        - containerPort: 8080
+        - containerPort: 9443
+            name: webhook-server
+            protocol: TCP
+        args:
+        - "--metrics-bind-address=:8082" # Metrics port changed to 8082
+```
```

Comment:
@andreyvelich Thank you for the suggestion. I've added a section on how to configure the default port for metrics exporting. Please let me know if you have further comments.